<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="OB-Yogesh" failures="1" tests="1" name="test.PartialReconciledCases.ScriptVATReturn1stVATReturnReport_SToFRS_DefaultProfile" time="31.734" errors="0" timestamp="2021-03-18T20:26:40 IST" skipped="0">
  <testcase classname="test.PartialReconciledCases.ScriptVATReturn1stVATReturnReport_SToFRS_DefaultProfile" name="GoToUrlAssertUrlPositiveEnterUserNamePositiveEnterPasswordClickLoginButtonGoToUrlAssertUrlCl" time="31.734">
    <failure message="Expecting URL - http://sandbox4.nomismasolution.co.uk/AccountUI/VATReturn.aspx?FYCode=52455&amp;CompanyCode=13259 Found http://sandbox4.nomismasolution.co.uk/AccountUI/AccessDenied.aspx?invalid=N" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Expecting URL - http://sandbox4.nomismasolution.co.uk/AccountUI/VATReturn.aspx?FYCode=52455&CompanyCode=13259 Found http://sandbox4.nomismasolution.co.uk/AccountUI/AccessDenied.aspx?invalid=N
at org.testng.Assert.fail(Assert.java:99)
at pages.VATReturn1stVATReturnReport.AssertUrl(VATReturn1stVATReturnReport.java:67)
at test.PartialReconciledCases.ScriptVATReturn1stVATReturnReport_SToFRS_DefaultProfile.GoToUrlAssertUrlPositiveEnterUserNamePositiveEnterPasswordClickLoginButtonGoToUrlAssertUrlCl(ScriptVATReturn1stVATReturnReport_SToFRS_DefaultProfile.java:116)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- GoToUrlAssertUrlPositiveEnterUserNamePositiveEnterPasswordClickLoginButtonGoToUrlAssertUrlCl -->
  <system-out/>
</testsuite> <!-- test.PartialReconciledCases.ScriptVATReturn1stVATReturnReport_SToFRS_DefaultProfile -->
